cowprotocol/solver_dashboard/active_solvers_1372857.sql

```diff
@@ -0,0 +1,41 @@
+-- This query finds all the active (i.e. whitelisted) solvers
+-- First we find the firest settlement for each solver
+-- Then we fet the environment and name of the solver for the prod and barn environments
+-- Finally we display the name of the solver with both of its addresses
+-- Note: if a solver has x barn address and y prod address, we will display x*y rows for that solver
```

Comment:
Good point. We might need to rethink that, although the goal is to have at most one address per solver per environment, or drop the distinction between prod and barn (for solvers) altogether.

I think i am fine with this, for now, although already today Barter has multiple and unnecessary entries because of that colocated and non-colocated addresses being whitelisted at the same time
https://dune.com/queries/1372857